aten/src/ATen/native/native_functions.yaml

```diff
@@ -2583,12 +2585,14 @@
   dispatch:
     CPU: mm_cpu
     CUDA: mm_cuda
+    MkldnnCPU: mkldnn_mm
```

Comment:
I don't think we need stretch MkldnnCPU into mm or mm.out, etc. This is going to bring more user level issues (not supported MkldnnCPU blahlblah...) for adjacent OPs.